.readthedocs.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+
+build:
+  image: latest
+
+python:
+  version: 3.7
+  install:
+    - method: pip
+      path: .
+      extra_requirements:
+        - docs
+        - all
```

Comment:
I don't think you have an ``all`` extras_require so this last line can be removed.